cmd/cdi-controller/controller.go

```diff
@@ -256,18 +256,17 @@ func start(cfg *rest.Config, stopCh <-chan struct{}) {
 		}
 	}()
 
-	go func() {
-		err = configController.Run(1, stopCh)
-		if err != nil {
-			klog.Fatalf("Error running config controller: %+v", err)
-		}
-	}()
-
 	startSmartController(extClient, csiInformerFactory, smartCloneController, stopCh)
+
+	if err := mgr.Start(signals.SetupSignalHandler()); err != nil {
```

Comment:
signal handlers are set elsewhere in this file as well.  Should prob only do it in one place.  I think moving leader election and signal handling to controller runtime may make sense.

Reply:
Until everything is converted, we will have some duality and duplication between the two mechanisms, one of which is this, as we move controllers that will solve itself.

Reply:
I am open to suggestions here, not sure how to make it cleaner.